graviti/utility/type.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""."""
+
+from typing import AbstractSet, Protocol, Tuple, TypeVar, Union
```

Comment:
the file name type is conflict with python builtin `type`,
maybe `typing.py` is a better choice.